### ISSUE TITLE
remove newlines from keyword fields

### ISSUE THREAD

**Issue by jduss4** [CDRH/data#104]
I cannot think of any situations where keywords should have return characters in them, since we are using them as exact matches / faceting.

Related to issues in API:  https://github.com/CDRH/api/pull/82
https://github.com/CDRH/api/pull/81

We have two options I see:
We could make this change across all keyword fields automatically and then if some project does need a newline in a keyword field, we can cross that bridge later with an override, etc.  Alternatively, we could add a line to each keyword field that smushes the contents, but then any overrides of those methods would need to imitate the behavior, as well.

**Comment by karindalziel** [comment]
I can't ever think of a situation we'd want a newline in a keyword, so I like option 1

**Comment by techgique** [comment]
Option 1 seems like a reasonable approach to me too. The ability to override later if it ever becomes necessary should cover our needs :)

**Comment by jduss4** [comment]
Agreed!  Let's plan on doing this next time datura comes up.  I do not believe any of the existing repos using datura in production or pseudo production (cather, whitman) will need overrides if return characters are removed
